openfpga/src/base/openfpga_bitstream_command_template.h

```diff
@@ -150,6 +150,18 @@ ShellCommandId add_build_fabric_bitstream_command_template(
   /* Add an option '--verbose' */
   shell_cmd.add_option("verbose", false, "Enable verbose output");
 
+  /* Add an option '--write_file' */
+  CommandOptionId opt_write_file = shell_cmd.add_option(
```

Comment:
Command ``build_fabric_bitstream`` does not consider any I/O. 
Instead ``write_fabric_bitstream`` does the writer job.
I suggest to add options to that command when using capn' proto.

https://openfpga.readthedocs.io/en/master/manual/openfpga_shell/openfpga_commands/fpga_bitstream_commands/#write-fabric-bitstream